axiom/nr_span_event.h

```diff
@@ -115,6 +145,8 @@ extern void nr_span_event_set_transaction_name(nr_span_event_t* event,
                                                const char* transaction_name);
 extern void nr_span_event_set_category(nr_span_event_t* event,
                                        nr_span_category_t category);
+extern void nr_span_event_set_spankind(nr_span_event_t* event,
+                                       nr_span_spankind_t category);
```

Comment:
Not that it matters much here, but just to avoid confusion in the future and help in searching through the code:
```suggestion
                                       nr_span_spankind_t spankind);
```

Reply:
Please see: https://github.com/newrelic/newrelic-php-agent/pull/991/commits/862604b081029206c2072275588076691d30560a